build.gradle

```diff
@@ -10,14 +10,16 @@ buildscript {
 
 plugins {
     id 'java-library'
+    id 'java-test-fixtures'
     id 'maven-publish'
     id 'com.jfrog.artifactory' version '4.13.0'
     id 'com.diffplug.gradle.spotless' version '3.27.2'
     id 'com.github.spotbugs' version '4.0.8'
 }
 
 group 'bio.terra'
-version '0.0.41-SNAPSHOT'
+// DO NOT SUBMIT
+version '0.1.0-SNAPSHOT'
```

Comment:
Don't forget this - I assume the maven local bit is for testing without publishing?

Reply:
Thanks. Yep.